pkg/engine/engine.go

```diff
@@ -51,7 +52,12 @@ func NewEngine(
 	ivCache imageverifycache.Client,
 	contextLoader engineapi.ContextLoaderFactory,
 	exceptionSelector engineapi.PolicyExceptionSelector,
+	isCluster *bool,
```

Comment:
Why use a pointer?

Reply:
I wasn't sure whether the value should be for some callers of this function true or false, so i wanted to create a default. I.E if the pointer is nil use a default of true.
And when clarified for all callers what boolean value they should use we can use a normal value not a pointer. An example of this is

`cmd/cli/kubectl-kyverno/processor/generate.go:116`
```golang
	c := generate.NewGenerateControllerWithOnlyClient(client, engine.NewEngine(
		cfg,
		config.NewDefaultMetricsConfiguration(),
		jmespath.New(cfg),
		adapters.Client(client),
		nil,
		imageverifycache.DisabledImageVerifyCache(),
		store.ContextLoaderFactory(s, nil),
		nil,
		nil, // unknown value that defaults to true
	))
```